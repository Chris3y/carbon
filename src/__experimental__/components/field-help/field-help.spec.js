import React from 'react';
import FieldHelp from './field-help.component';
