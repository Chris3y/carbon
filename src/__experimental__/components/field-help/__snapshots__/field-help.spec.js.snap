// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FieldHelp renders as expected 1`] = `
.c0 {
  color: inherit;
  display: block;
  margin-left: 0px;
  margin-right: 0px;
  margin-top: 8px;
  white-space: pre-wrap;
  width: 100%;
}

<span
  className="c0"
  data-element="help"
>
  help text
</span>
`;
